src/amd64/rfxencode_tile_amd64.c

```diff
@@ -53,7 +53,7 @@ rfx_encode_component_rlgr1_amd64_sse2(struct rfxencode *enc, const char *qtable,
     {
         return 1;
     }
-    *size = rfx_encode_diff_rlgr1(enc->dwt_buffer1, buffer, buffer_size);
+    *size = rfx_encode_diff_rlgr1(enc->dwt_buffer1, buffer, buffer_size, 64);
```

Comment:
Just curious, where is this 64 come from? How to calculate?

Reply:
Two steps are done, a diff and rlgr1.  LL3 is the band that gets diffed.  In the original RemoteFX, LL3 was 8x8.  In Progressive RemoteFX, LL3 is 9x9.  These should be defines.